AppCenter/AppCenter/MSAppCenter.h

```diff
@@ -83,6 +83,7 @@
 
 /**
  * Enable or disable the SDK as a whole. In addition to AppCenter resources, it will also enable or disable all registered services.
+ * The state is stored on disk, so it won't change in following app launches.
```

Comment:
Let me give it a try - "The state is persist on disk across app launches."

Reply:
Let's not use disk as already discussed on android pr and docs pr